kubetest/e2e/runner.go

```diff
@@ -30,6 +31,11 @@ import (
 	"k8s.io/test-infra/kubetest/process"
 )
 
+var (
+	e2eMasterOSDistribution = flag.String("e2e-master-os-distro", "", "master-os-distro flag to pass to e2e tests")
+	e2eNodeOSDistribution   = flag.String("e2e-node-os-distro", "", "node-os-distro flag to pass to e2e tests")
```

Comment:
I'm not really sure how we want these flags to work.  We could pass them into kops, and then kops could set them on the tester.  But that feels like we'll end up duplicating the flags for each installation tool.

It would be better if the e2e tests themselves didn't need this flag (they could easily just figure it out using SSH or a kubectl get nodes), but in the meantime...

Reply:
kubetest flags are a total mess :/

Reply:
maybe we can at least prefix provider specific flags? I don't think most other installers will have separate flags for master and node OS images...?